docker/README.md

```diff
@@ -1,51 +1,66 @@
-# api3/airnode-artifacts
+# api3/airnode-packaging
 
-This is a Docker container for building all the packages in the monorepo and providing their build artifacts. Other
-package specific containers can use this one to obtain files necessary for their run without building or installing any
-additional dependencies.
+This is a Docker container for building all the packages in the monorepo, publishing the NPM packages (TODO) and
+building the Docker containers.
+
+## How it works
+
+The container uses so called Docker-in-Docker method to build packages and Docker container in the clean Dockerized
+environment. The building flow looks like this:
+
+1. The local NPM registry is spun up
+2. Packages are built and published in the local NPM registry
+3. Docker containers are built, installing the packages from the local NPM registry
 
 ## Build
 
-Builds need to be run from the root directory to build the necessary context.
+In order to build Airnode packaging image run the following command from the root directory:
+
+```bash
+yarn docker:build:packaging
+```
 
 Usage of the new Docker Engine integrating BuildKit is highly recommended as it will both decrease the build time and
 allow conditional build steps. You can read more about how to enable it in its
 [documentation](https://docs.docker.com/develop/develop-images/build_enhancements/).
 
-### Local
+## Usage
 
-You can build the container image from your current workspace by providing a build type `local` (this is the default)
+In order for the container to work correctly you need to pass the Docker daemon socket from your system, so the
+containers can be used "within" the containers:
 
 ```bash
-docker build --build-arg build=local -f docker/Dockerfile -t api3/airnode-artifacts:latest .
-# or with no build argument
+... -v /var/run/docker.sock:/var/run/docker.sock ...
```

Comment:
This is cross platform right?

Reply:
Feel free to test it on Mac, it should work there. Not sure about Windows but I don't think being able to build the images there is as important as being able to run them there.

Reply:
Yeah, will try to test this today.

Reply:
Can we mention (or create an issue) that this is unusable on macOS due to docker copy being unusably slow?

Reply:
I've changed the way the files are copied (as we've discussed on Slack). Can you test it again?

Reply:
Sorry that it took so long. I can confirm this works, but I had to [add back some missing dependencies](https://github.com/api3dao/airnode/pull/1382/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcL15).

I have also a bit cleaner suggestion for copying the files. Essentially you can do a git clone from a local repo. See: https://github.com/api3dao/airnode/commit/47ddc74ad1be3a7248732a685d9e847fee14d991